src/Simulation/Native/src/simulator/wavefunction.hpp

```diff
@@ -549,6 +550,40 @@ class Wavefunction
         return kernels::jointprobability(wfn_, bs, get_qubit_positions(qs));
     }
 
+    /// \pre: Qubits, listed in `q`, must be unentangled and in state |0>.
```

Comment:
```suggestion
    /// \pre: Qubits, listed in `q`, must be unentangled and in the |00…0⟩ state.
```

[nit]: If `q` is in the |00…0⟩ state, that implies that they are unentangled. I agree it's still helpful to emphasize, though.

Reply:
My rational for phrasing the comment like this was that for entangled qubits it doesn't make sense to talk about their own state. The qubits also might not be adjacent in the computational basis. Would it be clearer to say "Each qubit, listed in `q` must be unentangled and in state |0>"?

Reply:
I think that makes sense, yeah. As written, the comment may be misunderstood as requiring that the register consist of a single qubit, I guess? All fairly minor in any case.